buildpack.md

```diff
@@ -486,15 +486,36 @@ If generated, this build BOM MUST contain all `bom` entries in each `build.toml`
 
 #### Layers
 
-A buildpack MAY create, modify, or delete `<layers>/<layer>/` directories and `<layers>/<layer>.toml` files as specified in the [Layer Types](#layer-types) section.
+##### Providing Layers
+A buildpack MAY create a new layer by creating a `<layers>/<layer>/` directories and `<layers>/<layer>.toml` file as specified in the [Layer Types](#layer-types) section.
+
+##### Reusing Layers
 
-To decide what layer operations are appropriate, the buildpack should consider:
+The lifecycle provides a mechanism for buildpacks to explicitly opt into reusing layers from a previous build. The buildpack may modify cached layers before reusing them.
+
+To decide whether layer reuse is appropriate, the buildpack should consider:
 
 - Whether files in the `<app>` directory have changed since the layer was created.
 - Whether the environment has changed since the layer was created.
 - Whether the buildpack version has changed since the layer was created.
 - Whether new application dependency versions have been made available since the layer was created.
 
+At the start of the build phase a buildpack MAY find:
+- Partial `<layers>/<layer>.toml` files describing layers from the previous builds. The restored Layer Content Metadata SHALL NOT contain `launch`, `build`, or `cache` booleans.
```

Comment:
Given that this might be surprising, maybe we should call-out that it "SHALL NOT" even if those values were set on a previous build.

Reply:
done